### ISSUE TITLE
Do not invoke update_localdb if provide with an option flag.

### ISSUE THREAD

**Issue by soulomoon** [tldr-pages/tldr-c-client#27]
Could we provide an option that:
While the option flag is on, when an command is not fount, do not invoke update_localdb.
So I could manually update my cache if I want to.

It would be really convenient in the under circumstances:
1 poor network situation.
2 trying to get several commands instruction.

then it would be possible for:
Manually update cache before trying out several command,
instead of having to invoke update_localdb every time if the command is not found on localdb
since poor network situation.

```shell
~/g/p/C/u/u/intro > time tldr lss
This page doesn't exist yet!
Submit new pages here: https://github.com/tldr-pages/tldr
tldr lss  0.02s user 0.01s system 0% cpu 4.425 total
```

**Comment by SethFalco** [comment]
Resolved by https://github.com/tldr-pages/tldr-c-client/pull/97

> To prevent `tldr` from automatically updating its database, set the environment variable `TLDR_AUTO_UPDATE_DISABLED`.
> 
> — https://github.com/tldr-pages/tldr-c-client#configuration
